src/main/archetype/pom.xml

```diff
@@ -603,18 +595,18 @@
             <dependency>
                 <groupId>org.mockito</groupId>
                 <artifactId>mockito-core</artifactId>
-                <version>2.23.4</version>
+                <version>2.25.1</version>
                 <scope>test</scope>
             </dependency>
             <dependency>
-                <groupId>junit-addons</groupId>
-                <artifactId>junit-addons</artifactId>
-                <version>1.4</version>
```

Comment:
I personally, use junit-addons pretty regularly, so I will have to get used to adding it back in. 😃   Still, if it's not used by the files in the archetype directly there's not a stron reason to keep it in.

On a personal note, the main reason I use it was because it is in this archetype.  I saw it there, wondered what it was used for, so I googled it and thought "wow, I can use that!".  I've been using it ever since.